ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -546,29 +568,35 @@ define([
       }
     },
 
-    isUtmLatLonValid: function(lat) {
-      return lat <= 84 && lat >= -80
+    isUtmUpsLatLonValid: function(lat) {
+      return lat !== undefined && lat >= -90 && lat <= 90
     },
 
-    // This method is called when the UTM point radius coordinates are changed by the user.
-    setRadiusUtm: function() {
-      if (this.isLocationTypeUtm()) {
-        if (this.isUtmPointRadiusDefined()) {
-          var utmParts = this.parseUtmPointRadius()
-          if (utmParts !== undefined) {
-            var utmResult = this.UTMtoLL(utmParts)
-
-            if (utmResult !== undefined) {
-              this.set(utmResult)
-
-              var usngsStr = converter.LLtoUSNG(
-                utmResult.lat,
-                utmResult.lon,
-                usngPrecision
-              )
-              this.set('usng', usngsStr, { silent: true })
+    // This method is called when the UTM/UPS point radius coordinates are changed by the user.
+    setRadiusUtmUps: function() {
+      if (this.isLocationTypeUtmUps()) {
```

Comment:
✏️ consider inverting predicates that have no alternate path to collapse part of this 'arrow'